test/VWT.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Parser = require('../lib')
+const chai = require('chai')
+const should = chai.Should()
+
+chai.use(require('chai-things'))
+
+describe('VWT', () => {
+  it('True wind converts ok', () => {
+    const delta = new Parser().parse('$IIVWT,030.,R,10.1,N,05.2,M,018.7,K*75')
+//'$IIMWV,074,T,05.85,N,A*2E'
+// $IIVWT,030.,R,10.1,N,05.2,M,018.7,K*75
```

Comment:
Why the commented lines?

Reply:
only overlooked to cleanup :-) 
deleted.